docker-entrypoint.sh

```diff
@@ -44,6 +44,13 @@ if [[ "$1" == "kong" ]]; then
   if [[ "$2" == "docker-start" ]]; then
     kong prepare -p "$PREFIX" "$@"
 
+    # remove all dangling sockets in $PREFIX dir before starting Kong
+    for localfile in "$PREFIX"/*; do
+      if [ -S $localfile ]; then
```

Comment:
Need to quote `$localfile` or use `[[` in this case:

```suggestion
      if [[ -S $localfile ]]; then
```